infrastructure/main.tf

```diff
@@ -32,4 +32,16 @@ module "case-management-web" {
     PRINT_SERVICE_URL_REMOTE = "${var.ccd_print_service_url}"
     WEBSITE_NODE_DEFAULT_VERSION = "8.9.4"
   }
+
+  module "ccd-case-management-web-vault" {
+    source              = "git@github.com:hmcts/moj-module-key-vault?ref=master"
+    name                = "ccd-case-management-web${var.env}" // Max 24 characters
```

Comment:
it needs to be max 24 chars including env name which varies between 3 and 7 chars.
`ccd-case-web-{var.env}` should do the trick